### ISSUE TITLE
data problem

### ISSUE THREAD

**Issue by Alice-Thirty** [JaidedAI/EasyOCR#781]
https://github.com/JaidedAI/EasyOCR/blob/master/custom_model.md
How can I label the image with the link?
Is it good ?
```
123.jpg THEFIRSTSTATE198606DELAWARE03
```
![image](https://user-images.githubusercontent.com/108393685/178735619-ea1a08b9-4b75-4919-9e7a-3038162cbfcf.png)

**Comment by rkcosmos** [comment]
no, it should be separated into 3 images. One image for each line.
